lib/src/streams/stream_handler.dart

```diff
@@ -95,6 +102,20 @@ class Http2StreamImpl extends TransportStream
       => _pushStreamFun(this, requestHeaders);
 
   void terminate() => _terminateStreamFun(this);
+
+  set onTerminated(void Function(int) handler) {
+    _onTerminated = handler;
+    if (_terminatedErrorCode != null && _onTerminated != null) {
+      _onTerminated(_terminatedErrorCode);
+    }
+  }
+
+  set cancelErrorCode(int errorCode) {
```

Comment:
Renamed to _handleTerminated (and made it a method). There was already a _terminatedErrorCode field, so I couldn't just make it private as is.